Cargo.lock

```diff
@@ -6046,7 +6048,7 @@ version = "0.1.9"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "cf221c93e13a30d793f7645a0e7762c55d169dbb0a49671918a2319d289b10bb"
 dependencies = [
- "windows-sys 0.59.0",
+ "windows-sys 0.48.0",
```

Comment:
Do we know why the versions for `windows-sys` changed here?

Reply:
Yeah this is pretty strange. I'd recommend checking out `Cargo.lock` from main, running `cargo check`, and seeing if these get reverted.

Reply:
Resolved